luigi/scheduler.py

```diff
@@ -1121,7 +1130,8 @@ def get_work(self, host=None, assistant=False, current_tasks=None, worker=None,
     @rpc_method(attempts=1)
     def ping(self, **kwargs):
         worker_id = kwargs['worker']
-        self._update_worker(worker_id)
+        worker = self._update_worker(worker_id)
+        return {"messages": worker.fetch_messages()}
```

Comment:
Looks like the [`@rpc_method`](https://github.com/spotify/luigi/blob/master/luigi/scheduler.py#L94) decorator uses [`RemoteScheduler._request`](https://github.com/spotify/luigi/blob/master/luigi/rpc.py#L141) which json-dumps the return value, so this should be fine.

If cross-language support is desired, JSON-RPC 2.0 specs would be awesome. https://github.com/riga/jsonrpyc (sorry for the self-promotion ;))